unconvert.go

```diff
@@ -292,12 +294,16 @@ func (noImporter) Import(path string) (*types.Package, error) {
 	panic("golang.org/x/tools/go/loader said this wouldn't be called")
 }
 
-func computeEdits(importPaths []string, os, arch string, cgoEnabled bool) fileToEditSet {
+func computeEdits(importPaths []string, os, arch string, cgoEnabled bool, buildTags string) fileToEditSet {
 	ctxt := build.Default
 	ctxt.GOOS = os
 	ctxt.GOARCH = arch
 	ctxt.CgoEnabled = cgoEnabled
 
+	if 0 < len(buildTags) {
```

Comment:
FWIW, I [personally think](https://dmitri.shuralyov.com/idiomatic-go#empty-string-check) it's more readable (and common) to write this as `if buildTags != "" {`.

Reply:
Additionally, the `if` statement can be completely removed without changing behavior.

If `buildTags` is the empty string, then `strings.Fields` returns an empty slice, which is what `ctxt.BuildTags` is anyway. So the `if` statement is not neccessary.